src/client/public/heartbeat.ts

```diff
@@ -99,14 +99,25 @@ import { Authenticator } from '@microsoft/office-js-helpers';
             return;
         }
 
+        // Upon user trusting snippet, update in local storage
```

Comment:
This is duplicate code.  And it should belong in the same helper where you're getting the key from (in fact, probably no reason to export the key anymore, just have this be self-contained logic)